app/templates/views/organisations-admin/approve-service-decision.html

```diff
@@ -31,8 +31,8 @@
             "items": [
               {"disabled": cannot_approve},
               {
-                "hint": {"html": "Enter the reason for your decision. Notify will share this with the person that made the request to go live."},
-                "conditional": { "html": form.rejection_reason(param_extensions={ "label": { "classes": "govuk-visually-hidden" }, "hint": None }) }
+                "hint": {"html": "Notify will share this with the person that made the request."},
+                "conditional": { "html": form.rejection_reason(param_extensions={ "hint": None }) }
```

Comment:
@tombye I **think** this should place the text field label above the hint text.

```suggestion
                "conditional": { "html": form.rejection_reason(param_extensions={ "hint": {"html": "Notify will share this with the person that made the request."} }) }
```

Reply:
I used a variant of this suggestion in [the commit I added](https://github.com/alphagov/notifications-admin/pull/5236/commits/161095b73fc15cc1d8b14b59b91540926aae1872).